src/common/mfu_daos.c

```diff
@@ -1392,6 +1392,7 @@ static int mfu_daos_obj_sync_recx_array(
         (*iod).iod_nr    = number;
         (*iod).iod_recxs = recxs;
         (*iod).iod_size  = size;
+        (*iod).iod_name = *akey;
```

Comment:
@daltonbohning is there a reason this was added to the sync_ function instead of the serialization one?